versioned/persist/tx/src/main/java/org/projectnessie/versioned/persist/tx/TxDatabaseAdapter.java

```diff
@@ -254,6 +269,20 @@ public Hash merge(
     }
   }
 
+  private void commitRefLog(
```

Comment:
Nit: Move this method to the other new private methods for the ref-log below (try to keep pulic/protected implementation at the top - and private below)

Reply:
done